themes/Sublette-color-theme.json

```diff
@@ -355,21 +915,319 @@
             }
         },
         {
+            "name": "Python Exceptions",
             "scope": [
-                "variable.parameter"
+                "support.type.exception.python"
             ],
             "settings": {
-                "foreground": "#ccced0ff",
+                "foreground": "#5e7"
             }
         },
         {
-            "name": "Python Exceptions",
+            "name": "Python: Support.exception",
```

Comment:
This change duplicated right upper code.

Reply:
Sorry, it was my mistake.

Reply:
You are right. I should've check that